README.md

```diff
@@ -30,6 +30,18 @@ python ./evaluate.py -m mace_mobilenetv1_nchw -t float32 -k android --target="op
 ```
 Refer to the below instructions for running the `scripts/evaluate.py` script for more information
 
+Additionaly, the script `evaluate_dyn_models.py` could be used to tune and evaluate layers from dynamic models (ssd-resnet, yolov3, faster-rcnn). Below, you can see examples of ssd-resnet evaluation with SD 8 Gen 1 statistics:
+```
+# float16 compute, float16 accumulate
+python ./evaluate_dyn_models.py -m ssd -t float16 -k android --target="opencl --device=adreno" -l ./logs_gen1/resnet_float16.autotvm.log
```

Comment:
Probably the models in `evaluate_dyn_models.py` should have the same names as in `evaluate.py`. Also the name of the log file should have the correspondent model name. Take a look on the old log files.